platform/ios/src/MGLMapView.mm

```diff
@@ -1709,6 +1717,15 @@ - (void)handleRotateGesture:(UIRotationGestureRecognizer *)rotate
 {
     if ( ! self.isRotateEnabled) return;
 
+    if (MGLDegreesFromRadians(self.rotationBeforeThresholdMet) < self.rotationThresholdWhileZooming && self.isZooming) {
```

Comment:
Is this still needed?

Reply:
I think it is. The `newDegreesForRotationGesture:` method alone doesn't seem to delay the map rotation while zooming, and we don't want to set the rotation to `0` unless zooming is occurring.

Reply:
Perhaps testing for `self.isZooming` only if sufficient?

Reply:
We don't want to set `rotate.rotation` to `0` every time there is a zoom gesture. We only want to set it to `0` when the the map is being zoomed and the threshold hasn't been met.